src/ride/ride.c

```diff
@@ -995,7 +995,10 @@ void ride_remove_peeps(int rideIndex)
 			exitZ = ride->station_heights[stationIndex];
 			mapElement = ride_get_station_exit_element(ride, exitX, exitY, exitZ);
 
-			exitDirection = mapElement->type & 3;
+			if (mapElement != NULL) {
+				exitDirection = mapElement->type;
+			}
+			exitDirection &= 3;
```

Comment:
exitDirection is 255 to start off with, maybe there should be some reason (better value for exitDirection) and warning for when there is no exit.